common/src/main/java/com/wynntils/mc/utils/RenderUtils.java

```diff
@@ -219,9 +227,9 @@ public static void drawTooltip(List<ClientTooltipComponent> lines, PoseStack pos
         int tooltipY = 4;
         // somewhat hacky solution to get around transparency issues - these colors were chosen to best match
         // how tooltips are displayed in-game
-        int backgroundColor = 0xFF100010;
-        int borderColorStart = 0xFF25005B;
-        int borderColorEnd = 0xFF180033;
+        CustomColor backgroundColor = CustomColor.fromInt(0xFF100010);
```

Comment:
I used to have a style checker that complained "No magic numbers in code". It was too annoying, so I turned it off, but it had a good point. Can you make these named private consts instead?